### PR TITLE
Add arrive@2.3.1 w/ git auto-update and some old versions

### PR DESCRIPTION
**Pull request by extend1994**
Pull request for issue: #10740 

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/uzairfarooq/arrive
 * NPM package url (optional): https://www.npmjs.com/package/arrive
 * License and its reference: [MIT](https://github.com/uzairfarooq/arrive/blob/master/LICENSE)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 13
   - Count of stars: 308
   - Count of forks: 45
 * NPM download stats (optional):
   - Downloads in the last day: 96
   - Downloads in the last week: 694
   - Downloads in the last month: 2,476

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/arrive/package.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
+  "name": "arrive",
+  "filename": "minified/arrive.min.js",
+  "version": "2.3.1",
+  "description": "arrive.js provides events to watch for DOM elements creation and removal. It makes use of Mutation Observers internally.",
+  "repository": {
+    "type": "git",
+    "url": "git://git@github.com/uzairfarooq/arrive.git"
```
Comment (PeterDaveHello) [inline ajax/libs/arrive/package.json]: Just a reminder, `git` protocol on GitHub with `git`(@) user info here won't work, if it comes from upstream, you can send a pull request to help them fix it.

File: ajax/libs/arrive/package.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  "author": "Uzair Farooq <uzair.farooq@outlook.com>",
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/uzairfarooq/arrive.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "minified/arrive*"
```
Comment (PeterDaveHello) [inline ajax/libs/arrive/package.json]: `minified` should better not exist in the path but basePath

File: ajax/libs/arrive/package.json [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    "javascript",
+    "js",
+    "watch",
+    "dynamic",
+    "creation",
+    "new",
+    "element",
+    "insertion",
+    "listen"
+  ],
```
Comment (sufuf3) [inline ajax/libs/arrive/package.json]: How about adding more keywords? such as `"DOM"`, `"removal"`, `"injected"`.
ref: https://github.com/uzairfarooq/arrive/blob/master/bower.json#L9-L13

File: ajax/libs/arrive/package.json (hunk 1 above)
Comment (extend1994) [inline ajax/libs/arrive/package.json]: Thanks for reminder.
Are git@github.com:uzairfarooq/arrive.git and https://github.com/uzairfarooq/arrive.git both okay?

Comment (sufuf3) [inline ajax/libs/arrive/package.json]: How about `git+https://github.com/uzairfarooq/arrive.git`?

I can use `git clone git://github.com/uzairfarooq/arrive.git` and `git clone https://github.com/uzairfarooq/arrive.git` to clone the repo.

But I can't clone the repo via `git clone git://git@github.com/uzairfarooq/arrive.git`

Comment (PeterDaveHello) [inline ajax/libs/arrive/package.json]: We usually don't need to touch this, if we need to do, as this case, just simply use `https://github.com/uzairfarooq/arrive.git`

**Comment by dogancelik** [comment]
Thanks for this! 😊

**Comment by extend1994** [comment]
@cdnjs/intern3 Please help me review this pull request, thank you!

**Comment by sufuf3** [comment]
ping @hare1039 @clairetsai818

**Comment by extend1994** [comment]
ping @hare1039 @clairetsai818

**Comment by PeterDaveHello** [comment]
@hare1039 so you should help rebase the branch with the latest master branch, shouldn't you?

**Pull request merged**